backend/Dockerfile

```
@@ -0,0 +1,33 @@
+FROM python:3.10 AS os-base
+
+RUN curl -sSL https://install.python-poetry.org | python3 -
+ENV PATH="/root/.local/bin:$PATH"
+RUN poetry config virtualenvs.create false
+RUN apt-get remove -y curl
+
+FROM os-base AS app-base
+
+WORKDIR /backend
+COPY ./src ./src
+COPY pyproject.toml .
+COPY gunicorn.conf.py .
```

As below, we can combine these copies if we want to:
```suggestion
COPY src pyproject.toml gunicorn.conf.py ./
```